doc/nrf/libraries/zigbee/shell.rst

```diff
@@ -29,6 +29,7 @@ To configure the Zigbee shell library, use the following options:
 * :kconfig:option:`CONFIG_ZIGBEE_SHELL_LOG_LEVEL`
 * :kconfig:option:`CONFIG_ZIGBEE_SHELL_CTX_MGR_ENTRIES_NBR`
 * :kconfig:option:`CONFIG_ZIGBEE_SHELL_MONITOR_CACHE_SIZE`
+* :kconfig:option:`CONFIG_ZIGBEE_SHELL_ZCL_CMD_TIMEOUT`
```

Comment:
Nothing to be added on the Configuring Zigbee libraries page?
https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/ug_zigbee_configuring_libraries.html#configuring-zigbee-shell

Reply:
I don't think this option is necessary for the `To extend a sample with the Zigbee shell command support, set the following Kconfig options:`. It's more or less used for fine-tuning the Zigbee Shell.